.licenserc.json

```diff
@@ -1,30 +1,32 @@
 [
-    {
-	"**/*.{go,proto}": [
-	    "Licensed under the Apache License, Version 2.0 (the \"License\");",
-	    "you may not use this file except in compliance with the License.",
-	    "You may obtain a copy of the License at",
-	    "    http://www.apache.org/licenses/LICENSE-2.0",
-	    "Unless required by applicable law or agreed to in writing, software",
-	    "distributed under the License is distributed on an \"AS IS\" BASIS,",
-	    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
-	    "See the License for the specific language governing permissions and",
-	    "limitations under the License."
-	],
- 	"ignore": [
-	    "vendor/"
-	]
-    },
-    {
-	"**/*.{go,proto}": [
-	    "SPDX-License-Identifier: Apache-2.0",
-	    "Copyright Contributors to the Submariner project."
-	],
-	"ignore": [
-	    "vendor/"
-	]
-    },
-    {
-	"LICENSE": "Apache License"
-    }
-]
+	{
+		"**/*.{go,proto}": [
+			"Licensed under the Apache License, Version 2.0 (the \"License\");",
+			"you may not use this file except in compliance with the License.",
+			"You may obtain a copy of the License at",
+			"    http://www.apache.org/licenses/LICENSE-2.0",
+			"Unless required by applicable law or agreed to in writing, software",
+			"distributed under the License is distributed on an \"AS IS\" BASIS,",
+			"WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+			"See the License for the specific language governing permissions and",
+			"limitations under the License."
+		],
+		"ignore": [
+			"vendor/"
+		]
+	},
+	{
+		"**/*.{go,proto}": [
+			"SPDX-License-Identifier: Apache-2.0",
+			"Copyright Contributors to the Submariner project."
+		],
+		"ignore": [
+			"pkg/client/",
+			"pkg/apis/lighthouse.submariner.io/v1alpha1/zz_generated.deepcopy.go",
+			"vendor/"
+		]
+	},
+	{
+		"LICENSE": "Apache License"
+	}
+]
```

Comment:
Was this change intentional?

Reply:
nope, auto formating with ctrl-s, thought I should leave it as vs code formated it but I'll revert